test/geocel/data/lar-sphere.org.json

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
I think with #1181 we don't need a separate ORANGE input file :) since the tests using it all require Geant4.

Reply:
Nice!

Reply:
Oh actually I do see test failures without it:
```console
[ RUN      ] LArSpherePreGenTest.host
unknown file: Failure
C++ exception with description "/home/runner/work/celeritas/celeritas/test/Test.cc:40:
celeritas: runtime error: std::ifstream(path).good() failed:
    Failed to open test data file at '/home/runner/work/celeritas/celeritas/test/geocel/data/lar-sphere.org.json'" thrown in the test body.
```

Reply:
Aha, I forgot single-precision mode doesn't yet support Geant4 conversion (mostly out of laziness). Your call whether to restore the file or put a `CELERITAS_TEST_IF_DOUBLE`/or `fixme_single` or whatever.

Reply:
Ah gotcha... I'll just revert and we can remove it again once that's supported.